src/NServiceBus.AcceptanceTests/Routing/When_multi_subscribing_to_a_polymorphic_event_on_unicast_transports.cs

```diff
@@ -54,14 +52,18 @@ public class Publisher1 : EndpointConfigurationBuilder
         {
             public Publisher1()
             {
-                EndpointSetup<DefaultPublisher>(b => b.OnEndpointSubscribed<Context>((args, context) =>
-               {
-                   context.AddTrace("Publisher1 OnEndpointSubscribed " + args.MessageType);
-                   if (args.MessageType.Contains(typeof(IMyEvent).Name))
-                   {
-                       context.Publisher1HasASubscriberForIMyEvent = true;
-                   }
-               }));
+                EndpointSetup<DefaultPublisher>(b =>
+                {
+                    b.OnEndpointSubscribed<Context>((args, context) =>
+                    {
+                        context.AddTrace("Publisher1 OnEndpointSubscribed " + args.MessageType);
+                        if (args.MessageType.Contains(typeof(IMyEvent).Name))
+                        {
+                            context.Publisher1HasASubscriberForIMyEvent = true;
+                        }
+                    });
+                    b.EnableFeature<FirstLevelRetries>();
+                }).WithConfig<TransportConfig>(c => { c.MaxRetries = 2; }); //Because subscription storages can throw on concurrecy violation and need to retry
```

Comment:
Why not just leave the default and move the comment

> //Because subscription storages can throw on concurrency violation and need to retry

to the line where you `EnableFeature<FirstLevelRetries>()`